data_structures/linked_list/middle_element_of_linked_list.py

```diff
@@ -0,0 +1,56 @@
+class Node:
+    def __init__(self, data:int) -> int:  
+        self.data = data  
+        self.next = None
+
+class LinkedList:
+    def __init__(self): 
+        self.head = None
+
+    def push(self, new_data:int) -> int:
+        new_node = Node(new_data) 
+        new_node.next = self.head 
+        self.head = new_node
+        return self.head.data
+
+    def middle_element(self) -> int:
+        '''
+        >>> link = LinkedList()
+        >>> link.push(5)
+        5
+        >>> link.push(6)
+        6
+        >>> link.push(8)
+        8
+        >>> link.push(8)
+        8
+        >>> link.push(10)
+        10
+        >>> link.push(12)
+        12
+        >>> link.push(17)
+        17
+        >>> link.push(7)
+        7
+        >>> link.push(3)
+        3
+        >>> link.middle_element()
+        10
+        >>> 
```

Comment:
Please add the following lines an then I thing we are ready to merge.
```suggestion
        >>> link.push(20)
        20
        >>> link.push(-20)
        -20
        >>> LinkedList.middle_element()
        12
```